src/Aspire.Hosting.PostgreSQL/PostgresContainerImageTags.cs

```diff
@@ -7,10 +7,10 @@ internal static class PostgresContainerImageTags
 {
     public const string Registry = "docker.io";
     public const string Image = "library/postgres";
-    public const string Tag = "16.2";
+    public const string Tag = "16.4";
```

Comment:
I don't think this tag exists? https://hub.docker.com/_/postgres/tags?page=2&page_size=&ordering=&name=16.4
![image](https://github.com/user-attachments/assets/cd88bab6-bbb0-41b6-b74e-7abdf83e8bd9)

Reply:
Oh nevermind it shows up if you search for "16." but not "16.4" 🤷

Reply:
blugh, I blame the docker hub search page. I had it on page 2 of results but it doesn't reset the page number when you change the search term. Move along, nothing to see here.